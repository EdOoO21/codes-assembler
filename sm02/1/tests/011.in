773 146 754 892 599 -47 -75 40 751 202 -612 -622 646 48 -26 289 257 624 -619 -808 -86 -379 -710 -815 103 658 823 420 299 -915 219 -189 975 -73 339 513 260 331 -678 276 -970 703 82 -871 -879 -927 -611 801 -505 228 -939 593 -50 -332 -98 210 725
