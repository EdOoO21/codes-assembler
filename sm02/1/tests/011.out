725
210
-98
-332
-50
593
-939
228
-505
801
-611
-927
-879
-871
82
703
-970
276
-678
331
260
513
339
-73
975
-189
219
-915
299
420
823
658
103
-815
-710
-379
-86
-808
-619
624
257
289
-26
48
646
-622
-612
202
751
40
-75
-47
599
892
754
146
773
